TROUBLESHOOTING.md

```diff
@@ -50,3 +50,52 @@ Then, make sure that `clang` is able to find it by adding the following line to
 ```
 export LIBRARY_PATH=$LIBRARY_PATH:/usr/local/opt/openssl/lib/
 ```
+
+## Issues running pipenv install --dev
```

Comment:
Do you mind searching this section with the one above? The seem to be about the same theme: Issues installing the Python MySQL package. The one above seems to be addressing issues with OSX and this one addresses with Ubuntu so maybe make one section for each OS?